autrainer/datasets/msppodcast.py

```diff
@@ -0,0 +1,202 @@
+                batch_size. Defaults to None.
+            train_transform: Transform to apply to the training set.
+                Defaults to None.
+            dev_transform: Transform to apply to the development set.
+                Defaults to None.
+            test_transform: Transform to apply to the test set.
+                Defaults to None.
+            stratify: Columns to stratify the dataset on. Defaults to None.
+            standardize: Whether to standardize the data. Defaults to False.
+            categories: used to filter out specific emotional categories. 
+                Useful for training on subset of data/classes, such as the classic
+                ["A", "H", "N", "S"] 4-class problem found in literature. 
+                Defaults to None.
+        """
+        # TODO: expand for multitask regression
+        assert target_column in ("EmoClass", "EmoAct", "EmoVal", "EmoDom"), (
+            f"{target_column} not included in target list, please choose one of: "
+            "[EmoClass, EmoAct, EmoVal, EmoDom]"
+        )
+        task = "classification" if target_column == "EmoClass" else "regression"
+        self.categories = categories
+        super().__init__(
+            task=task,
+            path=path,
+            features_subdir=features_subdir,
+            seed=seed,
+            metrics=metrics,
+            tracking_metric=tracking_metric,
+            index_column=index_column,
+            target_column=target_column,
+            file_type=file_type,
+            file_handler=file_handler,
+            batch_size=batch_size,
+            inference_batch_size=inference_batch_size,
+            train_transform=train_transform,
+            dev_transform=dev_transform,
+            test_transform=test_transform,
+            stratify=stratify,
+        )
+        self.standardize = standardize
+        if self.standardize:
+            train_data = torch.cat([x for x, *_ in self.train_dataset])
+            standardizer = Standardizer(
+                mean=train_data.mean(0).tolist(),
+                std=train_data.std(0).tolist(),
+            )
+
+            self.train_transform += standardizer
+            self.dev_transform += standardizer
+            self.test_transform += standardizer
+
+            self.train_dataset = self._init_dataset(
+                self.df_train, self.train_transform
+            )
+
+    @staticmethod
+    def download(path: str) -> None:  # pragma: no cover
+        """
+        Download the MSP-Podcast dataset.
+
+        As this dataset is not publicly-available, please download it manually
+        by contacting Prof. Carlos Busso: 
+        https://ecs.utdallas.edu/research/researchlabs/msp-lab/MSP-Podcast.html
+
+        This function will not do anything.
+
+        For more information on the data, see:
+        https://doi.org/10.1109/TAFFC.2017.2736999
+        """
+        return None
```

Comment:
The download method could be used to (ref [Structure and Loading](https://autrainer.github.io/autrainer/modules/datasets.html)) transform the data from the original state it is provided to our "opinionated" format, this could be similar to AIBO which is also private:
https://github.com/autrainer/autrainer/blob/4dd050d87e27e71c676da9788e2e4c5a29aaed17/autrainer/datasets/aibo.py#L124-L196

This would be required as we make the following assumptions:
- `${path}/default`: the default directory holds the raw files (audio)
- `${path}/*.csv`: the index (file paths) and targets are mapped through one or more dataframes

Reply:
Again, as above, this is for the `preprocessing` transform, right? And you would basically add instructions for the user and expect them to rename the `Audios` folder to `default`?

Reply:
The way to circumvent this is to add additional attributes to all dataset configurations specifying the paths of the default dataset subdirectory (relative to `${path}` or absolute) and providing a list of CSV paths in the same way.

Thats how completely customized paths could be used in theory, but i originally opted against this and "enforced" a predefined dataset structure as dataset configurations are already quite complicated and this would add even more things for users to consider and set up.